pkg/adapter/mtping/runner_test.go

```diff
@@ -300,7 +307,7 @@ func validateSent(t *testing.T, ce *adaptertesting.TestCloudEventsClient, wantDa
 		t.Errorf("Expected event with contentType=%q to be sent, got %q", wantContentType, gotContentType)
 	}
 
-	if got := event.Data(); string(got) != wantData {
+	if got := event.Data(); bytes.Compare(wantData, got) != 0 {
```

Comment:
@eclipselu can you address this warnings

Reply:
done